### ISSUE TITLE
Node v18.0.0

### ISSUE THREAD

**Issue by donavanbecker** [homebridge/homebridge#3114]
### Analysis

Not able to run with Node v18.0.0

### Expected Behavior

load successful

### Steps To Reproduce

Install Node 18.0.0 and then restart homebridge

### Logs

```markdown
AssertionError [ERR_ASSERTION]: Could not find valid addresses for interface 'anpi2'
    at NetworkManager.getCurrentNetworkInterfaces (/usr/local/lib/node_modules/homebridge/node_modules/@homebridge/ciao/src/NetworkManager.ts:419:13)
    at /usr/local/lib/node_modules/homebridge/node_modules/@homebridge/ciao/src/NetworkManager.ts:166:12
    at new Promise (<anonymous>)
    at new NetworkManager (/usr/local/lib/node_modules/homebridge/node_modules/@homebridge/ciao/src/NetworkManager.ts:165:24)
    at new MDNSServer (/usr/local/lib/node_modules/homebridge/node_modules/@homebridge/ciao/src/MDNSServer.ts:193:27)
    at new Responder (/usr/local/lib/node_modules/homebridge/node_modules/@homebridge/ciao/src/Responder.ts:129:19)
    at Function.getResponder (/usr/local/lib/node_modules/homebridge/node_modules/@homebridge/ciao/src/Responder.ts:121:25)
    at Object.getResponder (/usr/local/lib/node_modules/homebridge/node_modules/@homebridge/ciao/src/index.ts:52:20)
    at new CiaoAdvertiser (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Advertiser.ts:112:27)
    at Accessory.<anonymous> (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:1232:26)
```

### Configuration

```json
N/A
```

### Environment

- OS: macOS 12.4 beta 2
- Software: Homebridge 1.4.0
- Node: 18.0.0
- npm: 8.6.0

### Process Supervisor

hb-service

### Additional Context

Not a huge issue currently since it is easy to move back with hb-service, but figured I would post this to make people aware.

**Comment by bengalaviz** [comment]
I had this issue too on my raspberrypi after installing nodejs 18. I ended up disabling ipv6 on my pi and the error went away.

**Comment by oznu** [comment]
Thank you for reporting this, we will investigate.

Please note that Node.js v18 will not be officially supported until 2022-10-25.

https://github.com/homebridge/homebridge/wiki/How-To-Update-Node.js

**Comment by donavanbecker** [comment]
@oznu I should probably move this under Ciao?

**Comment by oznu** [comment]
Raise a seperate one. Otherwise we'll probably keep getting them here.

**Comment by giuliomagnifico** [comment]
Same issue here on RPi Zero W with DietPI. Updated node via dietpi-software to v18, after this the Homebridge server went up but all the devices were offline in Home app. 

Revert back to v16.14.2 (as suggested for the Zero W) and all was fine again.

**Comment by oznu** [comment]
Changes to provide preliminary support are in beta:

https://github.com/homebridge/homebridge/wiki/Homebridge-Beta-Testing

**Comment by oznu** [comment]
This particular issue with Node v18 should be fixed in 1.4.1.

**Comment by dppeak** [comment]
@oznu I just updated my Raspberry Pi to the latest Homebridge image and it's still on 16.18.0.  When I did the hb-service update-node it updated to 18.12.0.  I began to get many EHOSTUNREACH errors when trying to access registry.npmjs.org.   could be wrong, but I think it has something to do with the Pi image setup.
